lambdas/db-migration/src/migrations/20201014105054_create_pdrs_table.ts

```diff
@@ -3,17 +3,17 @@ import * as Knex from 'knex';
 export const up = async (knex: Knex): Promise<void> =>
   knex.schema.createTable('pdrs', (table) => {
     table
-      .increments('cumulusId')
+      .increments('cumulus_id')
       .comment('Internal Cumulus ID for a PDR')
       .primary();
     table
-      .integer('collectionCumulusId')
-      .references('cumulusId')
+      .integer('collection_cumulus_id')
```

Comment:
These changes need to be propagated to the PDR write in the `sf-event-sqs-to-db-records` lambda

Reply:
There are also a couple places in the Lambda that refer to `cumulusId` and not `cumulus_id`

Reply:
I think I've fixed those instances where it interacts with knex. The places where it is not directly interfacing with knex I've left camel case because ESLint complains.

Reply:
I'm not sure to what we extent we want to change that across the board. It's not an ideal situation.

Reply:
> The places where it is not directly interfacing with knex I've left camel case because ESLint complains.

Do you mean variable names? I prefer those in camelcase if so. I don't think it's a big deal for column names and variable names to be different

Reply:
Yes, variable names. I guess the other cases are object keys, so they are treated differently.